bin/fastly_nsq

```diff
@@ -1,7 +1,11 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
-require 'pry'
+begin
+  require 'pry'
+rescue LoadError
+  STDOUT.puts 'Pry not available. Not loading Pry.'
```

Comment:
Why do we even care? seems like we can just remove the `require` altogether no?

Reply:
we don't and you are correct